nss-day-cohort-39/tabloid-full-stack-st-clairs
# Requesting to merge post list into master

### Changes
- Added several files in both the API in Visual Studio and the clientside in VSC to successfully render the list of posts where they are approved and was published before the current time.

### Testing
- Make sure that when the link labeled Posts is clicked that it shows a list of posts. 
- Make sure that there are only around 23 posts. This is the number after they have been filtered.
